### PR TITLE
readme: readme make over

### PR DESCRIPTION
**Pull request by fanminshi**

### CODE REVIEW

**Comment by fanminshi** [comment]
work in progress.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,42 @@
...
+### Manual
+
+Download latest jetcd jar from [releases](https://github.com/coreos/jetcd/releases) and all its dependent jars:
+
+[grpc-core-1.5.0.jar](http://repo1.maven.org/maven2/io/grpc/grpc-core/1.5.0/)
+[grpc-netty-1.5.0.jar](http://repo1.maven.org/maven2/io/grpc/grpc-netty/1.5.0/)
+[grpc-protobuf-1.5.0.jar](http://repo1.maven.org/maven2/io/grpc/grpc-protobuf/1.5.0/)
+[grpc-stub-1.5.0.jar](http://repo1.maven.org/maven2/io/grpc/grpc-stub/1.5.0/)
+[slf4j-api-1.7.2.jar](http://repo1.maven.org/maven2/org/apache/directory/studio/org.slf4j.api/1.7.2/)
+[slf4j-simple-1.7.25.jar](http://repo1.maven.org/maven2/org/slf4j/slf4j-simple/1.7.25/)
```
Comment (lburgazzoli) [inline README.md]: Include maven/gradle coordinate first, people usually rely on maven/gradle to automatically grab dependencies

Comment (lburgazzoli) [inline README.md]: this is not needed

File: README.md [hunk 2]
```diff
@@ -1,10 +1,42 @@
...
+  <artifactId>jetcd</artifactId>
+  <version>0.0.1</version>
+</dependency>
+```
+
+Development snapshots are available in [Sonatypes's snapshot repository](https://oss.sonatype.org/content/repositories/snapshots/).
+
+### Gradle
+
+TODO
```
Comment (lburgazzoli) [inline README.md]: dependencies {
    compile('com.coreos:jetcd:0.0.1')
}

File: README.md (hunk 1 above)
Comment (fanminshi) [inline README.md]: do you mean just download jar from release is suffient? or `slf4j-simple-1.7.25.jar` is not needed?

```
### Manual

Download latest jetcd jar from [releases](https://github.com/coreos/jetcd/releases) 

```

Comment (lburgazzoli) [inline README.md]: slf4j-simple is not required as it is one of the implementation of the slf4j API often used for testing & so but real application need better logging framework. If none is provided slf4j uses Nop logger which simple does not log

**Comment by fanminshi** [comment]
I'll examples section in README when https://github.com/coreos/jetcd/pull/214/ is merged.

**Comment by fanminshi** [comment]
All fixed. PTAL

**Pull request merged**